tests/backup/backup_test.go

```diff
@@ -147,10 +147,32 @@ var _ = Describe("{BasicBackupCreateWithRules}", func() {
 				CreateBackupLocation(provider, backup_location_name, BackupLocationUID, CredName, CloudCredUID, bucketName, orgID)
 			}
 		})
+		Step("Creating backup schedule policies", func() {
+			dash.Info("Creating backup schedule policy of type interval")
+			interval_schedule_policy_info := CreateIntervalSchedulePolicy(5, int64(1*time.Minute), 2)
+			interval_policy_status := Backupschedulepolicy("interval", uuid.New(), orgID, interval_schedule_policy_info)
+			dash.VerifyFatal(interval_policy_status, nil, "Creating interval schedule policy")
+
+			dash.Info("Creating backup schedule policy of type daily")
+			daily_schedule_policy_info := CreateDailySchedulePolicy(1, "9:00AM", 2)
+			daily_policy_status := Backupschedulepolicy("daily", uuid.New(), orgID, daily_schedule_policy_info)
+			dash.VerifyFatal(daily_policy_status, nil, "Creating daily schedule policy")
+
+			dash.Info("Creating backup schedule policy of type weekly")
+			weekly_schedule_policy_info := CreateWeeklySchedulePolicy(1, "Friday", "9:10AM", 2)
```

Comment:
change the type to Weekday

Reply:
Done!!
http://aetos.pwx.purestorage.com/resultSet/testSetID/6971